src/main/java/org/broadinstitute/hellbender/tools/walkers/groundtruth/GroundTruthScorer.java

```diff
@@ -37,6 +37,99 @@
 import java.util.*;
 import java.util.zip.GZIPOutputStream;
 
+/**
+ * Converts Ultima reads into flow-based annotation, and provides some general statistics regarding
+ * quality and errors relative to the reference. Ultima data is flow-based, and thus the original computed
+ * quality refers to each flow, rather than each base. In the Ultima cram/bam, there is a per-base representation
+ * of the original flow qualities, where the original quality is distributed along each flow (homopolymer).
+ * In order to reconstitute the original flow information, the tool incorporates the information encoded
+ * in the Ultima cram/bam, and outputs both the read in flow space, as well as a conversion of the aligned
+ * reference portion into flow space, and an alignment score.
+ *
+ * <h3> Input </h3>
+ * <ul>
+ *     <li> Ultima aligned SAM/BAM/CRAM </li>
+ * </ul>
+ *
+ * <h3> Output </h3>
+ * <ul>
+ *     <li>Per read ground truth information CSV and a ground truth scoring quality report, in GATK report format</li>
+ * </ul>
+ *
+ * <h3>CSV Output Description </h3>
+ * csv with the read representation in flow space.  The csv includes the following columns:
+ * <li>ReadName</li>
+ * <li>ReadKey : The signal of the read at each flow according to the flow order</li>
+ * <li>ReadIsReversed : Whether the read is reversed in the alignment</li>
+ * <li>ReadMQ : The mapping quality of the read</li>
+ * <li>ReadRQ : The read rsq value</li>
```

Comment:
can you define RSQ here?

Reply:
> can you define RSQ here?

Actually a typo - should have been "rq" (the "rq" attribute of the read). Fixed